.changeset/small-pots-rest.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit/reactive-element': patch
+'lit': patch
+'lit-element': patch
+---
```

Comment:
This is a new feature so maybe it warrants a minor bump rather than patch?
```suggestion
---
'@lit/reactive-element': minor
'lit': minor
'lit-element': minor
---
```